### PR TITLE
JDF-835: Replace JBoss AS -> JBoss EAP. Replace Arquillian jbossas wi…

### PR DESCRIPTION
**Pull request by sgilda**
…th wildfly

### CODE REVIEW

File: bean-validation-custom-constraint/pom.xml [hunk 1]
```diff
@@ -237,15 +237,15 @@
...
             <plugins>
-                <!-- The Maven Surefire plugin tests your application. Here we ensure we are using a version compatible with Arquillian -->
+                <!-- The Maven Surefire plug-in tests your application. Here we ensure we are using a version compatible with Arquillian -->
                 <plugin>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>${version.surefire.plugin}</version>
                 </plugin>
-                <!-- The JBoss AS plugin deploys your war to a local JBoss EAP container -->
+                <!-- The WildFly plug-in deploys your war to a local JBoss EAP container -->
                 <!-- To use, set the JBOSS_HOME environment variable and run:
```
Comment (rafabene) [inline bean-validation-custom-constraint/pom.xml:249]: Just a doubt: Do we still using JBOSS_HOME even for EAP and WildFly in the docs ?

File: ejb-in-ear/ear/pom.xml [hunk 2]
```diff
@@ -60,12 +60,12 @@
     <build>
       <finalName>${project.parent.artifactId}</finalName>
         <plugins>
-            <!-- Ear plugin -->
+            <!-- Ear plug-in -->
```
Comment (rafabene) [inline ejb-in-ear/ear/pom.xml]: EAR (just to be fair enough with other changes :)

File: bean-validation-custom-constraint/pom.xml (hunk 1 above)
Comment (sgilda) [inline bean-validation-custom-constraint/pom.xml:249]: No one has told me whether that has changed. We use EAP_HOME as the replaceable command variable, but as far as I know, JBOSS_HOME is still the environment variable.

**Comment by rafabene** [comment]
You made good catches on this PR. It's good to go!

**Comment by sgilda** [comment]
Thanks @rafabene !

**Pull request merged**